spec/models/feedback_spec.rb

```diff
@@ -40,7 +40,7 @@
     end
     let(:data) do
       {
-        'feedback' => {
+        feedback: {
           '0' => { 'answer' => 'Answer0' },
```

Comment:
Shouldn't the data here be an array? It should try to mimic as closely as possible to real data (could potentially use the const you created?)

Reply:
yes! just doubled check in the design docs and yes, It should be an array of objects. But does this work better for querying questions?